contracts/invariant.ral

```diff
@@ -143,31 +148,43 @@ Contract Invariant(
         addPool{caller -> ALPH: mapEntryDeposit * 4}(caller, poolKey, initSqrtPrice, initTick, timestamp, config.admin)
     }
 
-    @using(preapprovedAssets = true)
+    @using(preapprovedAssets = true, updateFields = true)
     pub fn addFeeTier(feeTier: FeeTier) -> () {
         let caller = callerAddress!()
         checkCaller!(caller == config.admin, InvariantError.NotAdmin)
 
+        assert!(feeTierCount != MaxFeeTiers, InvariantError.FeeTierLimitReached)
```

Comment:
A check like this would convey the relationship between these variables better. 
```suggestion
        assert!(feeTierCount < MaxFeeTiers, InvariantError.FeeTierLimitReached)
```